src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetUptime.cs

```diff
@@ -36,16 +36,15 @@ protected override void ProcessRecord()
             {
                 TimeSpan uptime = TimeSpan.FromSeconds(Stopwatch.GetTimestamp() / Stopwatch.Frequency);
 
-                switch (ParameterSetName)
+                if (Since.ToBool())
                 {
-                    case TimespanParameterSet:
-                        // return TimeSpan of time since the system started up
-                        WriteObject(uptime);
-                        break;
-                    case SinceParameterSet:
-                        // return Datetime when the system started up
-                        WriteObject(DateTime.Now.Subtract(uptime));
-                        break;
+                    // return Datetime when the system started up
+                    WriteObject(DateTime.Now.Subtract(uptime));
```

Comment:
Th comment is obvious. It is better to point parameter set in comment.

Reply:
I didn't write that comment. I just altered the lines surrounding it. Do you want to expand the scope of the PR to general code improvements as well?

Reply:
Just to be clear, I don't mean that sarcastically. If you'd like me to polish up the code there I'm happy to do it. Just, as a baseline, it feels out of scope to me. 🙂

Reply:
These comments do not add value. Therefore, I would prefer to delete them.

Reply:
> It is better to point parameter set in comment.

Do you mean, by this, that rather than delete the comment, it would be better to change it from "this is what I'm doing" to "this is why I'm doing it"?

Reply:
I think the code is quite simple and short and requires no comments. (But if you want to add which branch belongs to which set of parameters, then you can do it.)